apps/sailfish/qml/components/PlayerControls.qml

```diff
@@ -76,6 +108,11 @@ Row {
         icon.height: 75; icon.width: 75
         height: 75; width: 75
         enabled: !!player
-        onClicked: playerControls.player.skipNext()
+        onClicked: {
+            if (settings.hapticsEnabled) {
+                rumbleEffect.start(2);
+            }
+            playerControls.player.skipNext()
+        }
```

Comment:
https://github.com/mzanetti/kodimote/blob/master/apps/ubuntu/qml/ItemDetails.qml#L86 :scream:

It works, don't touch :smiley:

Nah, if @accumulator wants to change them, feel free to do so, otherwise I'll probably change them when I'm in the mood (which is probably when I'm updating the Harbour branch, as QtFeedback still isn't allowed, and I'll thus need to remove it everywhere again)